src/esp/sensor/Sensor.h

```diff
@@ -37,11 +37,16 @@ enum class ObservationSpaceType {
   TEXT = 2,
 };
 
+enum class SensorSubtype {
+  PINHOLE = 0,
+  ORHTOGRAPHIC = 1,
+};
+
 // Specifies the configuration parameters of a sensor
 struct SensorSpec {
   std::string uuid = "rgba_camera";
   SensorType sensorType = SensorType::COLOR;
-  std::string sensorSubtype = "pinhole";
+  SensorSubtype sensorSubtype = SensorSubtype::PINHOLE;
```

Comment:
Please note the addition of this enum to describe camera type.  Originally a string holding "pinhole" it did not seem to be used anywhere.

Reply:
This seems more like CAMERA_TYPE / PROJECTION_TYPE than sensor subtype

Reply:
Just using the previous terminology - I wanted to keep the semantic changes to a minimum.